Makefile.am

```diff
@@ -209,6 +209,13 @@ multi-word: po/cockpit.pot
 title2sentence: multi-word
 	tools/title2sentence.py -i titles -o t2s.sh
 
+RSYNC_HOST ?= c
+RSYNC_DEST ?= $(RSYNC_HOST):/
+rsync:
+	$(MAKE)
+	$(MAKE) install DESTDIR=tmp/rsync >/dev/null
+	rsync -avcO tmp/rsync/ $(RSYNC_DEST)
```

Comment:
Why the `-c`, isn't that more expensive? Are you worried about different timezones on the target or so? (I would have thought that it would compare seconds-since-epoch Unix timestamps)

I had to look up `-c` and `-O`. For scripts, let's please use long options.

Reply:
`rsync` needs to be added to `.PHONY`.

Reply:
If we don't use `--checksum` then we end up copying every file every time, because the version in the `DESTDIR` was just copied there a moment ago and is therefore newer than the target.

I adjusted to long options and added `.PHONY`.